app-text/apvlv/apvlv-0.4.0.ebuild

```
@@ -0,0 +1,27 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit xdg cmake
+
+DESCRIPTION="apvlv is a PDF/EPUB Viewer and its behaviour like Vim"
```

We don't usually repeat package name in DESCRIPTION. This looks better:
`SET(CPACK_PACKAGE_DESCRIPTION_SUMMARY "apvlv - Alf's PDF/DJVU/EPUB Viewer like Vim")`
```suggestion
DESCRIPTION="Alf's PDF/DJVU/EPUB Viewer like Vim"
```
which is also closer to the current description.